internal/controller/model_pod.go

```diff
@@ -71,9 +81,96 @@ func (g *PodLoadBalancerModelGenerator) findPort(subset *corev1.EndpointSubset,
 	return -1, errPortNotFoundInSubset
 }
 
+func containsPort(port int32, proto *corev1.Protocol, portList []networkingv1.NetworkPolicyPort) bool {
+	klog.Infof("Looking for port %d proto %#v", port, *proto)
```

Comment:
We might want to use a lower level than Info here, because that's going to be very verbose otherwise.

Reply:
I deleted containsPort as it was not needed anymore and changed some debug messages to `klog.V(1)`.